server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel4.java

```diff
@@ -252,6 +251,7 @@ protected void configureLevel() {
       ChangeLogLevelStandaloneService.class);
 
     add(
+      LogServerId.class,
       PluginDownloader.class,
       DeprecatedViews.class,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "DeprecatedViews"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)